### DISCUSSION TITLE
Remove Pnp Search and restore OOTB Search Box web part

### DISCUSSION THREAD

**Discussion by danielagent** [microsoft-search/pnp-modern-search#3683] [General]
I am new to custom web parts and recently installed the PnP Modern Search (v4) web part following the steps provided - download SFPX package, connect API's etc. It has since replaced the original OOTB Search Box webpart that was available previously.

We would like to revert back to the original state with the OOTB Search Box web part again. I have attempted to remove the PnP search web part by removing the App from the App Catalogue in SharePoint Admin center. Also remove from App Catalog Recycle Bin. I have not removed the API's prompted as part of PnP installation instructions.

**Comment by wobba** [comment]
The OOB search box which was available for a short while was a mistake and it has been redacted. It was never planned to be released.
